saleor/graphql/plugins/mutations.py

```diff
@@ -45,6 +49,7 @@ class Meta:
     @classmethod
     def perform_mutation(cls, root, info, **data):
         plugin_id = data.get("id")
+        channel_slug = data.get("channel")
         data = data.get("input")
         manager = info.context.plugins
         plugin = manager.get_plugin(plugin_id)
```

Comment:
Shouldn't we have here the same channel validation as it was added to `PaymentInitialize`?

Reply:
No. We still have some plugins which are configurable globally (like Emails, Webhooks). PaymentPlugins by default are configurable by channel. 
So here, it is fine if I will receive the channel as a None value

Reply:
Slightly off-topic, but wouldn't it make sense to configure transactional emails on a per-channel basis?

Reply:
We can convert them to per channel but I would prefer to do that in separate PR